install/update_95_xx.php

```diff
@@ -49,6 +49,51 @@ function update95toXX() {
    require __DIR__ . '/update_95_xx/domains.php';
    require __DIR__ . '/update_95_xx/devicebattery.php';
    require __DIR__ . '/update_95_xx/reservationitem.php';
+   require __DIR__ . '/update_95_xx/recurrentchange.php';
+
+   //add is_fqdn on some domain records types
```

Comment:
This should probably not be here

Reply:
Indeed, error during rebase.
I've rebased again to fix conflicts, should be good now.